resources/openapi/yaml/registration-service.yaml

```diff
@@ -39,18 +39,10 @@ components:
       properties:
         did:
           type: string
-        name:
-          type: string
         status:
           type: string
           enum:
           - ONBOARDING_INITIATED
           - AUTHORIZING
           - AUTHORIZED
           - DENIED
-        supportedProtocols:
-          type: array
-          items:
-            type: string
```

Comment:
Are the `supportedProtocols` realated to the service URLs and therefor removed here?

Reply:
yes, from now on all the data related to participant (name, idsUrl, supportedProtocols etc) should be resolved from the didDocument which is an identity of the participant